sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -2614,6 +2614,7 @@ piEnqueueKernelLaunch(pi_queue Queue, pi_kernel Kernel, pi_uint32 WorkDim,
   assert(Kernel);
   assert(Queue);
   assert((WorkDim > 0) && (WorkDim < 4));
+  assert(GlobalWorkOffset[0] == 0);
```

Comment:
Why don't you check the other dimensions too? And also it can be NULL (actually I expect it to be NULL most of times). See here for the semantics of this argument: https://www.khronos.org/registry/OpenCL/sdk/2.2/docs/man/html/clEnqueueNDRangeKernel.html